iceoryx_posh/cmake/iceoryx_posh_deployment.hpp.in

```diff
@@ -46,6 +47,21 @@ constexpr uint64_t IOX_MAX_PUBLISHER_HISTORY = static_cast<uint32_t>(@IOX_MAX_PU
 constexpr uint32_t IOX_MAX_CHUNKS_HELD_PER_SUBSCRIBER_SIMULTANEOUSLY =
     static_cast<uint32_t>(@IOX_MAX_CHUNKS_HELD_PER_SUBSCRIBER_SIMULTANEOUSLY@);
  constexpr uint32_t IOX_MAX_NUMBER_OF_NOTIFIERS = static_cast<uint32_t>(@IOX_INTERNAL_MAX_NUMBER_OF_NOTIFIERS@);
+ constexpr uint32_t IOX_MAX_PROCESS_NUMBER = static_cast<uint32_t>(@IOX_MAX_PROCESS_NUMBER@);
```

Comment:
Correct me if I am wrong, but these appear all to be new options according to the diff. But I was sure we e.g. already had some of these for say request-response in sme way (e.g. `IOX_MAX_CLIENTS_PER_SERVER`).

Can you confirm that these are new parameters?